helm/idol-find/Chart.yaml

```
@@ -10,8 +10,8 @@
 # END COPYRIGHT NOTICE
 apiVersion: v2
 name: idol-find
-version: "0.3.2"
-appVersion: "23.4.0"
+version: "0.3.3"
```

Since this is a fairly big change:
```suggestion
version: "0.4.0"
```